### DISCUSSION TITLE
Two different private areas, one app

### DISCUSSION THREAD

**Discussion by amesas** [nextauthjs/next-auth#6280] [Help] [answered]
### Question 💬

1) We have to different private areas, for example: /customers /admins
2) Each area uses its own Provider: EmailProvider, CredentialsProvider

How do we setup two different api routes? /api/auth-customers/ /api/auth-admins/?

I guess using different NEXTAUTH_URL, but how do we force each Provider to use different env variables?

### How to reproduce ☕️

No link needed.

### Contributing 🙌🏽

No, I am afraid I cannot help regarding this

**Comment by adeturner** [comment, marked as answer]
No expert here but just to possibly unblock you (unless you really do need two separate domains, in which case I think the negative answer is found in #600)

I would setup two sets of providers in authoptions, one custom group called "admin*", one custom group called "customer*".  Your login page would invoke either signIn(admin) or signIn(customer) depending on the type of user.  You might decide which provider based on email domain, or some other lookup. I wrote some code in #5892 which might help.

**Reply by amesas** [reply]
@adeturner it worked for me :)

**Comment by amesas** [comment]
Thnx I will take a look! 

In my scenario is same domain, same next app, just two paths with different user base each path.
